chromadb/api/models/Collection.py

```diff
@@ -49,11 +52,15 @@ def __init__(
         name: str,
         id: UUID,
         embedding_function: Optional[EmbeddingFunction] = ef.DefaultEmbeddingFunction(),
+        tenant: Optional[str] = DEFAULT_TENANT,
+        database: Optional[str] = DEFAULT_DATABASE,
         metadata: Optional[CollectionMetadata] = None,
     ):
         super().__init__(name=name, metadata=metadata, id=id)
```

Comment:
I think we should follow the pattern of the previous fields - by passing it into __init__ and letting pydantic track and mange these fields for us.

Reply:
Agreed, BaseAPI implementers should map this.